charts/agent/values.yaml

```diff
@@ -36,6 +36,7 @@ config:
 
 ################################################
 deployment-cluster-events:
+  enabled: true
```

Comment:
i think these should go into the top level config vs. in here. also, im curious about the `config` top level field, isn't this all "config"? 

I think this field specifically should be `cluster.entities.enabled` or `cluster.resources.enabled`